data-raw/process_data_set_jessica.R

```diff
@@ -0,0 +1,18 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+nba_all_elo <- read_csv("data-raw/nba-elo/nbaallelo.csv")%>%
+  arrange(game_id, `_iscopy`)%>%
+  mutate_if(is.character,as.factor)%>%
+  mutate(opp_win_equiv = lead(win_equiv), 
+         opp_seasongame = lead(seasongame, 1),
+         date_game = mdy(date_game), 
+         is_playoffs = as.logical(is_playoffs),
+         notes =  as.character(notes))%>%
+  filter(`_iscopy` == 0)%>%
+  select(-c(`_iscopy`, game_location))
+
+usethis::use_data(nba_all_elo, overwrite = TRUE)
```

Comment:
Don't forget to delete the old `nbaallelo` from `data/` since it's been replaced by `nba_all_elo`. When I run the package check, it's giving a heads up to this effect:
```
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘nbaallelo’
Undocumented data sets:
  ‘nbaallelo’
```
It's considered good practice to run package checks before requesting a review so that obvious errors like this one get caught.